lib/Jsrt/ChakraCore.h

```diff
@@ -130,6 +130,21 @@ typedef struct JsNativeFunctionInfo
 /// <returns>The result of the call, if any.</returns>
 typedef _Ret_maybenull_ JsValueRef(CHAKRA_CALLBACK * JsEnhancedNativeFunction)(_In_ JsValueRef callee, _In_ JsValueRef *arguments, _In_ unsigned short argumentCount, _In_ JsNativeFunctionInfo *info, _In_opt_ void *callbackState);
 
+/// <summary>
+///     A Promise Rejection Tracker callback.
+/// </summary>
+/// <remarks>
+///     The host can specify a promise rejection tracker callback in <c>JsSetHostPromiseRejetionTracker</c>.
+///     If a promise is rejected with no reactions or a reaction is added to a promise that was rejected
+///     before it had reactions by default nothing is done.
+///     A Promise Rejection Tracker callback may be set - which will then be called when this occurs.
+/// </remarks>
+/// <param name="promise">The promise object, represented as a JsValueRef.</param>
+/// <param name="resolution">The value/cause of the rejection, represented as a JsValueRef.</param>
```

Comment:
`resolution` should be renamed, to either `reason`--or if you want something more generic, `value`.  As far as promises go, "resolve" usually only refers to fulfillment (e.g. `Promise.resolve()` is never a rejection unless you specifically pass it a rejected promise) so the current name is maybe a bit confusing.